mobile-app/app/components/TokenDropdownButton.tsx

```diff
@@ -34,7 +34,7 @@ export function TokenDropdownButton(props: {
         "opacity-30": props.status === TokenDropdownButtonStatus.Disabled,
         "opacity-100": props.status !== TokenDropdownButtonStatus.Disabled,
       })}
-      style={tailwind("flex flex-row items-center rounded-xl-v2 px-3 py-2.5")}
+      style={tailwind("flex flex-row items-center rounded-xl px-3 py-2.5")}
```

Comment:
border radius still not right
```suggestion
      style={tailwind("flex flex-row items-center rounded-xl px-3 py-2.5")}
```

Reply:
updated border radius to rounded-lg-v2